ui/app/components/secret-engine/configure-aws.ts

```diff
@@ -72,38 +100,77 @@ export default class ConfigureAwsComponent extends Component<Args> {
     this.disableAccessType = wifAttributesSet || iamAttributesSet;
   }
 
-  @task
-  @waitFor
-  *save(event: Event): Generator<Promise<boolean | LeaseConfigModel | RootConfigModel>> {
-    event.preventDefault();
-    this.resetErrors();
-    const { leaseConfig, rootConfig } = this.args;
-    // Note: aws/root-config model does not have any validations
-    const isValid = this.validate(leaseConfig);
-    if (!isValid) return;
-    // Check if any of the models' attributes have changed.
-    // If no changes to either model, transition and notify user.
-    // If changes to either model, save the model(s) that changed and notify user.
-    // Note: "backend" dirties model state so explicity ignore it here.
-
-    const leaseAttrChanged =
-      Object.keys(leaseConfig.changedAttributes()).filter((item) => item !== 'backend').length > 0;
-    const rootAttrChanged =
-      Object.keys(rootConfig.changedAttributes()).filter((item) => item !== 'backend').length > 0;
-
-    if (!leaseAttrChanged && !rootAttrChanged) {
-      this.flashMessages.info('No changes detected.');
-      this.transition();
-    }
+  @action continueSubmitForm() {
```

Comment:
This is a little easier to follow how the interruption happens. Nice mix of solutions without the extra tracked property.